### DISCUSSION TITLE
Save local copy

### DISCUSSION THREAD

**Discussion by CyberSathya** [Stirling-Tools/Stirling-PDF#687] [Ideas]
It would be great, if there is an option to save a local copy of the output files for easy reference at a later point of time.

The saved files can be made available for 30 days locally, after which, they can be auto-flushed to save disk space.

**Comment by JSDA123** [comment]
Yes this would be very useful. For example if using the endpoint /api/v1/convert/render/html/pdf it would be very helpful to have a link to the rendered pdf in the response body so that file can be easily grabbed and used.

**Comment by Rolzzz** [comment]
I'd prefer this be an admin switch if possible... I foresee it taking up a whole bunch of space real quick
